langroid/language_models/azure_openai.py

```diff
@@ -29,7 +29,7 @@ class AzureConfig(OpenAIGPTConfig):
 
     api_key: str = ""  # CAUTION: set this ONLY via env var AZURE_OPENAI_API_KEY
     type: str = "azure"
-    api_version: str = "2023-05-15"
+    api_version: str = "2024-02-15-preview"
```

Comment:
good to update here, can be overwritten via the `AZURE_OPENAI_API_VERSION` in the `.env`